frontend/smart-kickers-game/src/components/Game/CurrentGameplay/CurrentGameplay.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import { Button } from '../../Button/Button';
 import GameResults from '../GameResults/GameResults.js';
+import { useNavigate } from 'react-router-dom';
+import useCurrentGameplay from '../../../hooks/useCurrentGameplay';
+
+function CurrentGameplay() {
+  const { isVisible, handleStartGame, handleResetGame, handleEndGame } = useCurrentGameplay();
+  const navigate = useNavigate();
 
-function CurrentGameplay({ blueScore, whiteScore, handleStartGame, handleResetGame, handleEndGame, isVisible }) {
   return (
     <div>
-      <GameResults blueScore={blueScore} whiteScore={whiteScore} isVisible={isVisible} />
+      <GameResults isVisible={isVisible} />
```

Comment:
Why isn't `GameResults` and the stuff below (when `!isVisible`) a separate route?

Reply:
We have changed it.